.github/workflows/rust.yml

```diff
@@ -49,14 +61,22 @@ jobs:
         with:
           profile: minimal
           toolchain: ${{ matrix.toolchain }}
-          target: ${{ matrix.target }}
+          target: ${{ steps.generate-target.outputs.triple }}
           override: true
 
+      - name: Run cargo build
+        if: ${{ matrix.arch == 'aarch64' }}
+        # don't compile tls with tls support
+        run: |
+          cargo install cross
```

Comment:
Why use cross instead of native build?

Reply:
Because `rustls` depends on `ring`, which contains some pregenerated assemblies, `cross` is an easier way to setup cross compile toolchains (which is needed to compile the assemblies.)